### PR TITLE
fix/delete-bookmark-by-tweet&owner

### PR DESCRIPTION
**Pull request by NikPaukov**

### CODE REVIEW

File: src/routes/bookmarks.route.ts [hunk 1]
```diff
@@ -29,7 +31,19 @@ bookmarksRouter
...
+      const { tweet } = req.body;
+      await deleteBookmarkByTweetAndOwner(tweet, req.user._id);
+      console.log(tweet);
+      console.log(req.user._id);
+      res.sendStatus(200);
+    }
+  );
 bookmarksRouter
   .route("/:id")
   .delete(isAuthorized, validation(deleteBookmarkById), async (req, res) => {
```
Comment (DavidKorystov) [inline src/routes/bookmarks.route.ts:47]: I think this route already unnecessary, bcs deleting by tweetId is enough.

**Review by DavidKorystov — CHANGES_REQUESTED** [review]

File: src/routes/bookmarks.route.ts (hunk 1 above)
Comment (DavidKorystov) [reply src/routes/bookmarks.route.ts:47]: @alexandrtovmach

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
you PR had problem.

image

 Clean all console.log

**Review by VitaliiDenysiuk — APPROVED** [review]
lgtm

**Pull request merged**